Build failure: dupeguru

### Nixpkgs version

- Unstable (25.11)

### Steps to reproduce

Dupeguru is failing to build with the following nix code

```nix
{
    home-manager.users.beatlink = { pkgs, ... }: {
        home = {
            packages = with pkgs; [ dupeguru ];
        };
    };
}


```

### Can Hydra reproduce this build failure?

Yes, Hydra can reproduce this build failure.

### Link to Hydra build job

https://hydra.nixos.org/build/301780596

### Relevant log output

```console
Jul 13 09:59:30 Odin nixos-upgrade-start[101022]: unpacking 'github:BeatLink/TechNet/2fd4779c4db2e356a5045d1e910eb5d073bb59cb' into the Git cache...
Jul 13 09:59:59 Odin nixos-upgrade-start[101022]: unpacking 'github:nix-community/home-manager/fc25398450cdab61af9654928dfef9d101f51140?narHash=sha256-d0d7y9gjv9HU0Amws6kHJucR8bxrQ0PfVKxt2ll5cfs%3D' into the Git cache...
Jul 13 10:00:01 Odin nixos-upgrade-start[101022]: unpacking 'github:wamserma/flake-programs-sqlite/dde8afca6c88d362caafa04fe5c4283d3a83025e?narHash=sha256-q6o/ekGAVaSCr/rf6ry188ItGXMEYbAONHURbVEB8Fw%3D' into the Git cache...
Jul 13 10:00:04 Odin nixos-upgrade-start[101021]: building the system configuration...
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]: these 21 derivations will be built:
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/cja26ahmlxw3bywv7a1zv7kg5acrrr41-options.json.drv
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/25ns2l8g0wiif84b25zkkgagkjjsvif8-home-configuration-reference-manpage.drv
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/3mlq8aklkv4f1bkns99br0r34zy2ixbn-dupeguru-4.3.1.drv
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/fxzk4d3hbxxs7yb0jv7i0nykivlbg3k2-home-manager-path.drv
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/xylxq6j0axmhpjgkn4kbh21r1ybpdz1v-hm_.configplankdock1launchers18io.lmms.LMMS.dockitem.drv
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/hfjiwjkp2lqm4vcl2bfxvdys4f2r84ki-home-manager-files.drv
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/jr9j9q4vsqdmz9ppqbpldlqdyyqxy4hw-hm-dconf.ini.drv
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/q82p69gi9b83nx4m86iyca5m2s08px64-activation-script.drv
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/xhjfxh9m3hyjdnqrw9mvw2fljqg6gmcs-home-manager-generation.drv
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/390hkq3svxd2zcmkywx8pp74r8xsg9bj-unit-home-manager-beatlink.service.drv
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/7d5d63sdrkiiskqrzfv43976jlgs721j-impermanence-persist-files.drv
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/g4c15qpvmimrfv0jwmls343h5scxqnlc-unbindOrUnlink-Storage-Apps-Fun-LMMS-home-beatlink-.lmmsrc.xml.drv
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/aw5khh00mxi901djbrkzrlasjdcr0cfb-unit-persist-Storage-Apps-Fun-LMMS-home-beatlink-.lmmsrc.xml.service.drv
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/kcg7wcrwim290hsjdv01chd35ycvqj37-home-manager-path.drv
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/h514hv104nzcbln8rfn2kx755cj1dfqj-activation-script.drv
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/dsf3sv8v54r4dnn9wgyzcfylvjy74vxj-home-manager-generation.drv
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/h5adap7ykq6i18dlnv09n17irkvgk8cn-unit-home-manager-root.service.drv
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/hmi7afaydp6v63kw951vypvs1qkq4r52-system-units.drv
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/7k9xs0rha3n648x7kk2qfbwffz6a9fzr-etc.drv
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/nypfzik2ghmk4yb3adq7ddl0jwqf0s2l-impermanence-run-create-directories.drv
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/pwlrxnf96q61jx3pkas849x3c2d6cd7i-nixos-system-Odin-25.11.20250708.9807714.drv
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]: these 56 paths will be fetched (48.22 MiB download, 308.33 MiB unpacked):
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/r9awyw4mg7zsp5f7a4w2b1ji3n5pri7x-carla-2.5.9
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/1l1lbzxa0dc4ax3d2ncgx1nmmpfy3h7f-ensure-newer-sources-hook
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/vwk609rla0v549n7dn0fzbwdl50rp5y3-fltk-1.3.8
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/w8zsm8pvj7h2wfh8i6q2q4drc3rvrapj-liblo-0.32
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/3s2wf2yq3lm0yyq6lykcj9iwajwsfm70-lmms-1.2.2
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/3n5g9c4bapwn6qay7rdzxb6j71kan2ix-pytest-check-hook
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/m86g4g8imf0gkajl1ayymzz1sfa5wmim-python-catch-conflicts-hook
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/0xpkaxxx4bxfvjkzsycsir3svgjabafd-python-imports-check-hook.sh
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/w98inlmsagll2f71wc8zll0mm2sqd63k-python-namespaces-hook.sh
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/sk4k4kiki52na6zn48zzj6laa7szy4c2-python-remove-bin-bytecode-hook
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/fi0pim9l63w5l7a6299m5c7bka46x1fj-python-remove-tests-dir-hook
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/0z30si9fzrffww4dws2wb11rslsrampp-python3.11-alabaster-1.0.0
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/zgp17xjlkj2ihsihmdalknqd94g4jqbd-python3.11-babel-2.17.0
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/yf2afba53cswf4zz1j0m3n9h8q6wqv64-python3.11-certifi-2025.04.26
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/rrsyrr4hddnh5rzcwi0ndc8z3k9q827g-python3.11-charset-normalizer-3.4.2
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/j98jr7g4i2hixn6fx9abkrd0qd7pzp3m-python3.11-dbus-python-1.4.0
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/zzwkskx75akajxshd9600vg8vmclwx0s-python3.11-dbus-python-1.4.0-dev
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/ylw80ik4g8k141hblsvw0scz9pxhcqnl-python3.11-docutils-0.21.2
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/3x2agr0iag8p6b0s4g3m9w6bm5ka24hy-python3.11-hsaudiotag3k-1.1.3.post1
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/a59qn4ilrnxrqhjv3kdr8rmcw2xjw29h-python3.11-idna-3.10
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/7hkm98pj25vl8y157kmpyskx1xwwiw61-python3.11-imagesize-1.4.1
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/g4i082hfdxqbxq8xv3d4hj25r9psfrsx-python3.11-iniconfig-2.1.0
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/ar7wddcprlg734s4ky24i4n7xjza0c66-python3.11-jinja2-3.1.6
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/fnn4w00rfxyyjwhp2591xk2fy7ksqdvb-python3.11-markupsafe-3.0.2
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/wxh27f2kk6byhfpinm1w4ls6g356lp3w-python3.11-mutagen-1.47.0
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/g1lqwh7wzhizfzb2ykd5a86r39anj2ab-python3.11-packaging-25.0
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/n42qza7icjhg8h4n395n5lrq1mzajj74-python3.11-pluggy-1.6.0
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/6mr6fbyrgh12gwqkp8z3c7lrl7kgsrlm-python3.11-polib-1.2.0
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/ay3rqkkv803mvwcpwxky5bkb2c0h81gd-python3.11-pygments-2.19.1
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/kvy3z425q9njv0d1xxh1s9crqi8nz7dz-python3.11-pyqt5-5.15.10
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/0c9yvyq98dk9cxg44vqzi392l898n9jb-python3.11-pyqt5-5.15.10-dev
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/kzxhjrkjg78nr1j1vqyj4ld031ywdn71-python3.11-pyqt5-sip-12.17.0
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/znq3qvdh9nmqxhqmn1rb9bvisx7pssa1-python3.11-pystemmer-2.2.0.1
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/b0sxj74mniqj6q8yp46diznlmxjr91kl-python3.11-pytest-8.3.5
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/jf56lng5vjjp3b41b038nh0hq4xcp90b-python3.11-requests-2.32.3
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/j3mn9z5y05x6vcwbb45diaim7lqq160h-python3.11-roman-numerals-py-3.1.0
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/rig20d917sc9kajl9g78w2s1z6xzn3c0-python3.11-semantic-version-2.10.0
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/1pvwf8wcn7cddwsikp9ywzkjdhz5743s-python3.11-send2trash-1.8.3
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/63r62p69dpgzzh4iivn8zr26n9adysdl-python3.11-setuptools-80.7.1
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/2l09ksmmr5cdrbbhal04d5f0wcrnaxws-python3.11-snowballstemmer-3.0.1
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/7mjwp20fck8q179yn310xm7z7wyrj54n-python3.11-sphinx-8.2.3
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/rcbsisc0rr8m09rk85gqj90y02jcxlm2-python3.11-sphinxcontrib-applehelp-2.0.0
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/6scjq952a9774c51dayrrkv6mpa2dx4c-python3.11-sphinxcontrib-devhelp-2.0.0
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/s5xy0xqngzia9idq5y2h49p4am2isk3k-python3.11-sphinxcontrib-htmlhelp-2.1.0
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/p9gpy6h0n75n4jjw8zmmhkw1sbl7g6hj-python3.11-sphinxcontrib-jsmath-1.0.1
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/4q526f9m5k6njnxqhis47gq5q550567b-python3.11-sphinxcontrib-qthelp-2.0.0
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/sdl8yw4368bh6hak8cxhgrz53h6ghfyk-python3.11-sphinxcontrib-serializinghtml-2.0.0
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/jv2bnz123xklpgmjz1yk7kgfih3p18wf-python3.11-sphinxcontrib-websupport-2.0.0
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/lnxqcbxiv7ia160y6mafhabyl3lkkgdh-python3.11-urllib3-2.4.0
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/p6n0qp0xi52bz0vjjjhaqn5aqpq681cm-python3.13-pyliblo3-0.16.3
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/lsfynrx2cr709mg244dhxgl3yzpj33yk-python3.13-pyqt5-5.15.10
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/llzpvmb3l4w06549sygmvmsdn7rmzjvv-python3.13-pyqt5-5.15.10-dev
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/4szhfb2j5abwpyki7q7qjxysiwmld739-python3.13-pyqt5-sip-12.17.0
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/i8gg4faabgqidkddjyrhl5qs6la4xqa5-python3.13-rdflib-7.1.4
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/law57bk59qmwn52wrv4gpk9sqcy9w5ml-source
Jul 13 10:00:23 Odin nixos-upgrade-start[101133]:   /nix/store/6l5xka4a4c6whdz174as0nwxj772wip2-wrap-python-hook
Jul 13 10:00:24 Odin nixos-upgrade-start[101133]: building '/nix/store/7d5d63sdrkiiskqrzfv43976jlgs721j-impermanence-persist-files.drv'...
Jul 13 10:00:24 Odin nixos-upgrade-start[101133]: building '/nix/store/nypfzik2ghmk4yb3adq7ddl0jwqf0s2l-impermanence-run-create-directories.drv'...
Jul 13 10:00:25 Odin nixos-upgrade-start[101133]: building '/nix/store/g4c15qpvmimrfv0jwmls343h5scxqnlc-unbindOrUnlink-Storage-Apps-Fun-LMMS-home-beatlink-.lmmsrc.xml.drv'...
Jul 13 10:00:26 Odin nixos-upgrade-start[101133]: building '/nix/store/jr9j9q4vsqdmz9ppqbpldlqdyyqxy4hw-hm-dconf.ini.drv'...
Jul 13 10:00:26 Odin nixos-upgrade-start[101133]: copying path '/nix/store/vwk609rla0v549n7dn0fzbwdl50rp5y3-fltk-1.3.8' from 'https://cache.nixos.org'...
Jul 13 10:00:26 Odin nixos-upgrade-start[101133]: copying path '/nix/store/sk4k4kiki52na6zn48zzj6laa7szy4c2-python-remove-bin-bytecode-hook' from 'https://cache.nixos.org'...
Jul 13 10:00:26 Odin nixos-upgrade-start[101133]: copying path '/nix/store/fi0pim9l63w5l7a6299m5c7bka46x1fj-python-remove-tests-dir-hook' from 'https://cache.nixos.org'...
Jul 13 10:00:27 Odin nixos-upgrade-start[101133]: copying path '/nix/store/law57bk59qmwn52wrv4gpk9sqcy9w5ml-source' from 'https://cache.nixos.org'...
Jul 13 10:00:27 Odin nixos-upgrade-start[101133]: copying path '/nix/store/w8zsm8pvj7h2wfh8i6q2q4drc3rvrapj-liblo-0.32' from 'https://cache.nixos.org'...
Jul 13 10:00:27 Odin nixos-upgrade-start[101133]: copying path '/nix/store/lsfynrx2cr709mg244dhxgl3yzpj33yk-python3.13-pyqt5-5.15.10' from 'https://cache.nixos.org'...
Jul 13 10:00:27 Odin nixos-upgrade-start[101133]: copying path '/nix/store/4szhfb2j5abwpyki7q7qjxysiwmld739-python3.13-pyqt5-sip-12.17.0' from 'https://cache.nixos.org'...
Jul 13 10:00:27 Odin nixos-upgrade-start[101133]: copying path '/nix/store/i8gg4faabgqidkddjyrhl5qs6la4xqa5-python3.13-rdflib-7.1.4' from 'https://cache.nixos.org'...
Jul 13 10:00:27 Odin nixos-upgrade-start[101133]: copying path '/nix/store/1l1lbzxa0dc4ax3d2ncgx1nmmpfy3h7f-ensure-newer-sources-hook' from 'https://cache.nixos.org'...
Jul 13 10:00:27 Odin nixos-upgrade-start[101133]: copying path '/nix/store/m86g4g8imf0gkajl1ayymzz1sfa5wmim-python-catch-conflicts-hook' from 'https://cache.nixos.org'...
Jul 13 10:00:27 Odin nixos-upgrade-start[101133]: copying path '/nix/store/0xpkaxxx4bxfvjkzsycsir3svgjabafd-python-imports-check-hook.sh' from 'https://cache.nixos.org'...
Jul 13 10:00:27 Odin nixos-upgrade-start[101133]: copying path '/nix/store/w98inlmsagll2f71wc8zll0mm2sqd63k-python-namespaces-hook.sh' from 'https://cache.nixos.org'...
Jul 13 10:00:27 Odin nixos-upgrade-start[101133]: copying path '/nix/store/3x2agr0iag8p6b0s4g3m9w6bm5ka24hy-python3.11-hsaudiotag3k-1.1.3.post1' from 'https://cache.nixos.org'...
Jul 13 10:00:27 Odin nixos-upgrade-start[101133]: copying path '/nix/store/wxh27f2kk6byhfpinm1w4ls6g356lp3w-python3.11-mutagen-1.47.0' from 'https://cache.nixos.org'...
Jul 13 10:00:27 Odin nixos-upgrade-start[101133]: copying path '/nix/store/6mr6fbyrgh12gwqkp8z3c7lrl7kgsrlm-python3.11-polib-1.2.0' from 'https://cache.nixos.org'...
Jul 13 10:00:27 Odin nixos-upgrade-start[101133]: copying path '/nix/store/kzxhjrkjg78nr1j1vqyj4ld031ywdn71-python3.11-pyqt5-sip-12.17.0' from 'https://cache.nixos.org'...
Jul 13 10:00:27 Odin nixos-upgrade-start[101133]: building '/nix/store/cja26ahmlxw3bywv7a1zv7kg5acrrr41-options.json.drv'...
Jul 13 10:00:28 Odin nixos-upgrade-start[101133]: copying path '/nix/store/0z30si9fzrffww4dws2wb11rslsrampp-python3.11-alabaster-1.0.0' from 'https://cache.nixos.org'...
Jul 13 10:00:28 Odin nixos-upgrade-start[101133]: copying path '/nix/store/zgp17xjlkj2ihsihmdalknqd94g4jqbd-python3.11-babel-2.17.0' from 'https://cache.nixos.org'...
Jul 13 10:00:28 Odin nixos-upgrade-start[101133]: copying path '/nix/store/yf2afba53cswf4zz1j0m3n9h8q6wqv64-python3.11-certifi-2025.04.26' from 'https://cache.nixos.org'...
Jul 13 10:00:28 Odin nixos-upgrade-start[101133]: copying path '/nix/store/rrsyrr4hddnh5rzcwi0ndc8z3k9q827g-python3.11-charset-normalizer-3.4.2' from 'https://cache.nixos.org'...
Jul 13 10:00:28 Odin nixos-upgrade-start[101133]: copying path '/nix/store/j98jr7g4i2hixn6fx9abkrd0qd7pzp3m-python3.11-dbus-python-1.4.0' from 'https://cache.nixos.org'...
Jul 13 10:00:28 Odin nixos-upgrade-start[101133]: copying path '/nix/store/ylw80ik4g8k141hblsvw0scz9pxhcqnl-python3.11-docutils-0.21.2' from 'https://cache.nixos.org'...
Jul 13 10:00:28 Odin nixos-upgrade-start[101133]: copying path '/nix/store/a59qn4ilrnxrqhjv3kdr8rmcw2xjw29h-python3.11-idna-3.10' from 'https://cache.nixos.org'...
Jul 13 10:00:28 Odin nixos-upgrade-start[101133]: copying path '/nix/store/7hkm98pj25vl8y157kmpyskx1xwwiw61-python3.11-imagesize-1.4.1' from 'https://cache.nixos.org'...
Jul 13 10:00:28 Odin nixos-upgrade-start[101133]: copying path '/nix/store/g4i082hfdxqbxq8xv3d4hj25r9psfrsx-python3.11-iniconfig-2.1.0' from 'https://cache.nixos.org'...
Jul 13 10:00:28 Odin nixos-upgrade-start[101133]: copying path '/nix/store/fnn4w00rfxyyjwhp2591xk2fy7ksqdvb-python3.11-markupsafe-3.0.2' from 'https://cache.nixos.org'...
Jul 13 10:00:28 Odin nixos-upgrade-start[101133]: copying path '/nix/store/n42qza7icjhg8h4n395n5lrq1mzajj74-python3.11-pluggy-1.6.0' from 'https://cache.nixos.org'...
Jul 13 10:00:28 Odin nixos-upgrade-start[101133]: copying path '/nix/store/g1lqwh7wzhizfzb2ykd5a86r39anj2ab-python3.11-packaging-25.0' from 'https://cache.nixos.org'...
Jul 13 10:00:28 Odin nixos-upgrade-start[101133]: copying path '/nix/store/ay3rqkkv803mvwcpwxky5bkb2c0h81gd-python3.11-pygments-2.19.1' from 'https://cache.nixos.org'...
Jul 13 10:00:28 Odin nixos-upgrade-start[101133]: building '/nix/store/aw5khh00mxi901djbrkzrlasjdcr0cfb-unit-persist-Storage-Apps-Fun-LMMS-home-beatlink-.lmmsrc.xml.service.drv'...
Jul 13 10:00:28 Odin nixos-upgrade-start[101133]: copying path '/nix/store/kvy3z425q9njv0d1xxh1s9crqi8nz7dz-python3.11-pyqt5-5.15.10' from 'https://cache.nixos.org'...
Jul 13 10:00:28 Odin nixos-upgrade-start[101133]: copying path '/nix/store/znq3qvdh9nmqxhqmn1rb9bvisx7pssa1-python3.11-pystemmer-2.2.0.1' from 'https://cache.nixos.org'...
Jul 13 10:00:28 Odin nixos-upgrade-start[101133]: building '/nix/store/25ns2l8g0wiif84b25zkkgagkjjsvif8-home-configuration-reference-manpage.drv'...
Jul 13 10:00:28 Odin nixos-upgrade-start[101133]: copying path '/nix/store/j3mn9z5y05x6vcwbb45diaim7lqq160h-python3.11-roman-numerals-py-3.1.0' from 'https://cache.nixos.org'...
Jul 13 10:00:28 Odin nixos-upgrade-start[101133]: copying path '/nix/store/rig20d917sc9kajl9g78w2s1z6xzn3c0-python3.11-semantic-version-2.10.0' from 'https://cache.nixos.org'...
Jul 13 10:00:28 Odin nixos-upgrade-start[101133]: copying path '/nix/store/1pvwf8wcn7cddwsikp9ywzkjdhz5743s-python3.11-send2trash-1.8.3' from 'https://cache.nixos.org'...
Jul 13 10:00:28 Odin nixos-upgrade-start[101133]: copying path '/nix/store/63r62p69dpgzzh4iivn8zr26n9adysdl-python3.11-setuptools-80.7.1' from 'https://cache.nixos.org'...
Jul 13 10:00:28 Odin nixos-upgrade-start[101133]: copying path '/nix/store/lnxqcbxiv7ia160y6mafhabyl3lkkgdh-python3.11-urllib3-2.4.0' from 'https://cache.nixos.org'...
Jul 13 10:00:28 Odin nixos-upgrade-start[101133]: copying path '/nix/store/6scjq952a9774c51dayrrkv6mpa2dx4c-python3.11-sphinxcontrib-devhelp-2.0.0' from 'https://cache.nixos.org'...
Jul 13 10:00:28 Odin nixos-upgrade-start[101133]: copying path '/nix/store/s5xy0xqngzia9idq5y2h49p4am2isk3k-python3.11-sphinxcontrib-htmlhelp-2.1.0' from 'https://cache.nixos.org'...
Jul 13 10:00:28 Odin nixos-upgrade-start[101133]: copying path '/nix/store/p9gpy6h0n75n4jjw8zmmhkw1sbl7g6hj-python3.11-sphinxcontrib-jsmath-1.0.1' from 'https://cache.nixos.org'...
Jul 13 10:00:28 Odin nixos-upgrade-start[101133]: copying path '/nix/store/4q526f9m5k6njnxqhis47gq5q550567b-python3.11-sphinxcontrib-qthelp-2.0.0' from 'https://cache.nixos.org'...
Jul 13 10:00:28 Odin nixos-upgrade-start[101133]: copying path '/nix/store/sdl8yw4368bh6hak8cxhgrz53h6ghfyk-python3.11-sphinxcontrib-serializinghtml-2.0.0' from 'https://cache.nixos.org'...
Jul 13 10:00:28 Odin nixos-upgrade-start[101133]: copying path '/nix/store/rcbsisc0rr8m09rk85gqj90y02jcxlm2-python3.11-sphinxcontrib-applehelp-2.0.0' from 'https://cache.nixos.org'...
Jul 13 10:00:28 Odin nixos-upgrade-start[101133]: copying path '/nix/store/zzwkskx75akajxshd9600vg8vmclwx0s-python3.11-dbus-python-1.4.0-dev' from 'https://cache.nixos.org'...
Jul 13 10:00:28 Odin nixos-upgrade-start[101133]: copying path '/nix/store/ar7wddcprlg734s4ky24i4n7xjza0c66-python3.11-jinja2-3.1.6' from 'https://cache.nixos.org'...
Jul 13 10:00:29 Odin nixos-upgrade-start[101133]: copying path '/nix/store/b0sxj74mniqj6q8yp46diznlmxjr91kl-python3.11-pytest-8.3.5' from 'https://cache.nixos.org'...
Jul 13 10:00:29 Odin nixos-upgrade-start[101133]: copying path '/nix/store/2l09ksmmr5cdrbbhal04d5f0wcrnaxws-python3.11-snowballstemmer-3.0.1' from 'https://cache.nixos.org'...
Jul 13 10:00:29 Odin nixos-upgrade-start[101133]: copying path '/nix/store/p6n0qp0xi52bz0vjjjhaqn5aqpq681cm-python3.13-pyliblo3-0.16.3' from 'https://cache.nixos.org'...
Jul 13 10:00:29 Odin nixos-upgrade-start[101133]: copying path '/nix/store/llzpvmb3l4w06549sygmvmsdn7rmzjvv-python3.13-pyqt5-5.15.10-dev' from 'https://cache.nixos.org'...
Jul 13 10:00:29 Odin nixos-upgrade-start[101133]: copying path '/nix/store/6l5xka4a4c6whdz174as0nwxj772wip2-wrap-python-hook' from 'https://cache.nixos.org'...
Jul 13 10:00:29 Odin nixos-upgrade-start[101133]: copying path '/nix/store/jf56lng5vjjp3b41b038nh0hq4xcp90b-python3.11-requests-2.32.3' from 'https://cache.nixos.org'...
Jul 13 10:00:29 Odin nixos-upgrade-start[101133]: copying path '/nix/store/jv2bnz123xklpgmjz1yk7kgfih3p18wf-python3.11-sphinxcontrib-websupport-2.0.0' from 'https://cache.nixos.org'...
Jul 13 10:00:29 Odin nixos-upgrade-start[101133]: copying path '/nix/store/r9awyw4mg7zsp5f7a4w2b1ji3n5pri7x-carla-2.5.9' from 'https://cache.nixos.org'...
Jul 13 10:00:29 Odin nixos-upgrade-start[101133]: copying path '/nix/store/3n5g9c4bapwn6qay7rdzxb6j71kan2ix-pytest-check-hook' from 'https://cache.nixos.org'...
Jul 13 10:00:29 Odin nixos-upgrade-start[101133]: copying path '/nix/store/7mjwp20fck8q179yn310xm7z7wyrj54n-python3.11-sphinx-8.2.3' from 'https://cache.nixos.org'...
Jul 13 10:00:29 Odin nixos-upgrade-start[101133]: copying path '/nix/store/0c9yvyq98dk9cxg44vqzi392l898n9jb-python3.11-pyqt5-5.15.10-dev' from 'https://cache.nixos.org'...
Jul 13 10:00:30 Odin nixos-upgrade-start[101133]: building '/nix/store/kcg7wcrwim290hsjdv01chd35ycvqj37-home-manager-path.drv'...
Jul 13 10:00:30 Odin nixos-upgrade-start[101133]: building '/nix/store/3mlq8aklkv4f1bkns99br0r34zy2ixbn-dupeguru-4.3.1.drv'...
Jul 13 10:00:30 Odin nixos-upgrade-start[101133]: home-manager-path> created 227 symlinks in user environment
Jul 13 10:00:30 Odin nixos-upgrade-start[101133]: copying path '/nix/store/3s2wf2yq3lm0yyq6lykcj9iwajwsfm70-lmms-1.2.2' from 'https://cache.nixos.org'...
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: building '/nix/store/h514hv104nzcbln8rfn2kx755cj1dfqj-activation-script.drv'...
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> Sourcing python-remove-tests-dir-hook
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> Sourcing python-catch-conflicts-hook.sh
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> Sourcing python-remove-bin-bytecode-hook.sh
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> Sourcing python-imports-check-hook.sh
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> Using pythonImportsCheckPhase
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> Sourcing python-namespaces-hook
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> Sourcing pytest-check-hook
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> Using pytestCheckPhase
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> Running phase: qtPreHook
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> Running phase: unpackPhase
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> unpacking source archive /nix/store/law57bk59qmwn52wrv4gpk9sqcy9w5ml-source
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> source root is source
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> setting SOURCE_DATE_EPOCH to timestamp 315619200 of file "source/win_version_info.temp"
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> Running phase: patchPhase
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> Running phase: updateAutotoolsGnuConfigScriptsPhase
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> Running phase: configurePhase
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> no configure script, doing nothing
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> Running phase: buildPhase
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> build flags: SHELL=/nix/store/00zrahbb32nzawrmv9sjxn36h7qk9vrs-bash-5.2p37/bin/bash PREFIX=/nix/store/lib0a5j82f02ipx04m8dc1jal6l0x62f-dupeguru-4.3.1 NO_VENV=1
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: building '/nix/store/dsf3sv8v54r4dnn9wgyzcfylvjy74vxj-home-manager-generation.drv'...
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> msgfmt -o locale/ar/LC_MESSAGES/columns.mo locale/ar/LC_MESSAGES/columns.po
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> msgfmt -o locale/ar/LC_MESSAGES/core.mo locale/ar/LC_MESSAGES/core.po
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> msgfmt -o locale/ar/LC_MESSAGES/ui.mo locale/ar/LC_MESSAGES/ui.po
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> msgfmt -o locale/cs/LC_MESSAGES/columns.mo locale/cs/LC_MESSAGES/columns.po
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> msgfmt -o locale/cs/LC_MESSAGES/core.mo locale/cs/LC_MESSAGES/core.po
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> msgfmt -o locale/cs/LC_MESSAGES/ui.mo locale/cs/LC_MESSAGES/ui.po
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> msgfmt -o locale/de/LC_MESSAGES/columns.mo locale/de/LC_MESSAGES/columns.po
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> msgfmt -o locale/de/LC_MESSAGES/core.mo locale/de/LC_MESSAGES/core.po
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> msgfmt -o locale/de/LC_MESSAGES/ui.mo locale/de/LC_MESSAGES/ui.po
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> msgfmt -o locale/el/LC_MESSAGES/columns.mo locale/el/LC_MESSAGES/columns.po
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> msgfmt -o locale/el/LC_MESSAGES/core.mo locale/el/LC_MESSAGES/core.po
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> msgfmt -o locale/el/LC_MESSAGES/ui.mo locale/el/LC_MESSAGES/ui.po
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> msgfmt -o locale/en/LC_MESSAGES/columns.mo locale/en/LC_MESSAGES/columns.po
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> msgfmt -o locale/en/LC_MESSAGES/core.mo locale/en/LC_MESSAGES/core.po
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> msgfmt -o locale/en/LC_MESSAGES/ui.mo locale/en/LC_MESSAGES/ui.po
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> msgfmt -o locale/es/LC_MESSAGES/columns.mo locale/es/LC_MESSAGES/columns.po
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> msgfmt -o locale/es/LC_MESSAGES/core.mo locale/es/LC_MESSAGES/core.po
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> msgfmt -o locale/es/LC_MESSAGES/ui.mo locale/es/LC_MESSAGES/ui.po
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> msgfmt -o locale/fr/LC_MESSAGES/columns.mo locale/fr/LC_MESSAGES/columns.po
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> msgfmt -o locale/fr/LC_MESSAGES/core.mo locale/fr/LC_MESSAGES/core.po
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> msgfmt -o locale/fr/LC_MESSAGES/ui.mo locale/fr/LC_MESSAGES/ui.po
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> msgfmt -o locale/hy/LC_MESSAGES/columns.mo locale/hy/LC_MESSAGES/columns.po
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> msgfmt -o locale/hy/LC_MESSAGES/core.mo locale/hy/LC_MESSAGES/core.po
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> msgfmt -o locale/hy/LC_MESSAGES/ui.mo locale/hy/LC_MESSAGES/ui.po
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> msgfmt -o locale/it/LC_MESSAGES/columns.mo locale/it/LC_MESSAGES/columns.po
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> msgfmt -o locale/it/LC_MESSAGES/core.mo locale/it/LC_MESSAGES/core.po
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> msgfmt -o locale/it/LC_MESSAGES/ui.mo locale/it/LC_MESSAGES/ui.po
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> msgfmt -o locale/ja/LC_MESSAGES/columns.mo locale/ja/LC_MESSAGES/columns.po
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> msgfmt -o locale/ja/LC_MESSAGES/core.mo locale/ja/LC_MESSAGES/core.po
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> msgfmt -o locale/ja/LC_MESSAGES/ui.mo locale/ja/LC_MESSAGES/ui.po
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> msgfmt -o locale/ko/LC_MESSAGES/columns.mo locale/ko/LC_MESSAGES/columns.po
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> msgfmt -o locale/ko/LC_MESSAGES/core.mo locale/ko/LC_MESSAGES/core.po
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> msgfmt -o locale/ko/LC_MESSAGES/ui.mo locale/ko/LC_MESSAGES/ui.po
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> msgfmt -o locale/ms/LC_MESSAGES/columns.mo locale/ms/LC_MESSAGES/columns.po
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> msgfmt -o locale/ms/LC_MESSAGES/core.mo locale/ms/LC_MESSAGES/core.po
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> msgfmt -o locale/ms/LC_MESSAGES/ui.mo locale/ms/LC_MESSAGES/ui.po
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> msgfmt -o locale/nl/LC_MESSAGES/columns.mo locale/nl/LC_MESSAGES/columns.po
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> msgfmt -o locale/nl/LC_MESSAGES/core.mo locale/nl/LC_MESSAGES/core.po
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> msgfmt -o locale/nl/LC_MESSAGES/ui.mo locale/nl/LC_MESSAGES/ui.po
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> msgfmt -o locale/pl_PL/LC_MESSAGES/columns.mo locale/pl_PL/LC_MESSAGES/columns.po
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> msgfmt -o locale/pl_PL/LC_MESSAGES/core.mo locale/pl_PL/LC_MESSAGES/core.po
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> msgfmt -o locale/pl_PL/LC_MESSAGES/ui.mo locale/pl_PL/LC_MESSAGES/ui.po
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> msgfmt -o locale/pt_BR/LC_MESSAGES/columns.mo locale/pt_BR/LC_MESSAGES/columns.po
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> msgfmt -o locale/pt_BR/LC_MESSAGES/core.mo locale/pt_BR/LC_MESSAGES/core.po
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> msgfmt -o locale/pt_BR/LC_MESSAGES/ui.mo locale/pt_BR/LC_MESSAGES/ui.po
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> msgfmt -o locale/ru/LC_MESSAGES/columns.mo locale/ru/LC_MESSAGES/columns.po
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> msgfmt -o locale/ru/LC_MESSAGES/core.mo locale/ru/LC_MESSAGES/core.po
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> msgfmt -o locale/ru/LC_MESSAGES/ui.mo locale/ru/LC_MESSAGES/ui.po
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> msgfmt -o locale/tr/LC_MESSAGES/columns.mo locale/tr/LC_MESSAGES/columns.po
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> msgfmt -o locale/tr/LC_MESSAGES/core.mo locale/tr/LC_MESSAGES/core.po
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> msgfmt -o locale/tr/LC_MESSAGES/ui.mo locale/tr/LC_MESSAGES/ui.po
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> msgfmt -o locale/uk/LC_MESSAGES/columns.mo locale/uk/LC_MESSAGES/columns.po
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> msgfmt -o locale/uk/LC_MESSAGES/core.mo locale/uk/LC_MESSAGES/core.po
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> msgfmt -o locale/uk/LC_MESSAGES/ui.mo locale/uk/LC_MESSAGES/ui.po
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> msgfmt -o locale/vi/LC_MESSAGES/columns.mo locale/vi/LC_MESSAGES/columns.po
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> msgfmt -o locale/vi/LC_MESSAGES/core.mo locale/vi/LC_MESSAGES/core.po
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> msgfmt -o locale/vi/LC_MESSAGES/ui.mo locale/vi/LC_MESSAGES/ui.po
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> msgfmt -o locale/zh_CN/LC_MESSAGES/columns.mo locale/zh_CN/LC_MESSAGES/columns.po
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> msgfmt -o locale/zh_CN/LC_MESSAGES/core.mo locale/zh_CN/LC_MESSAGES/core.po
Jul 13 10:00:31 Odin nixos-upgrade-start[101133]: dupeguru> msgfmt -o locale/zh_CN/LC_MESSAGES/ui.mo locale/zh_CN/LC_MESSAGES/ui.po
Jul 13 10:00:32 Odin nixos-upgrade-start[101133]: building '/nix/store/h5adap7ykq6i18dlnv09n17irkvgk8cn-unit-home-manager-root.service.drv'...
Jul 13 10:00:32 Odin nixos-upgrade-start[101133]: dupeguru> msgfmt -o locale/zh_TW/LC_MESSAGES/columns.mo locale/zh_TW/LC_MESSAGES/columns.po
Jul 13 10:00:32 Odin nixos-upgrade-start[101133]: dupeguru> msgfmt -o locale/zh_TW/LC_MESSAGES/core.mo locale/zh_TW/LC_MESSAGES/core.po
Jul 13 10:00:32 Odin nixos-upgrade-start[101133]: dupeguru> msgfmt -o locale/zh_TW/LC_MESSAGES/ui.mo locale/zh_TW/LC_MESSAGES/ui.po
Jul 13 10:00:32 Odin nixos-upgrade-start[101133]: dupeguru> python3 build.py --modules
Jul 13 10:00:32 Odin nixos-upgrade-start[101133]: dupeguru> Traceback (most recent call last):
Jul 13 10:00:32 Odin nixos-upgrade-start[101133]: dupeguru>   File "/build/source/build.py", line 13, in <module>
Jul 13 10:00:32 Odin nixos-upgrade-start[101133]: dupeguru>     from setuptools import sandbox
Jul 13 10:00:32 Odin nixos-upgrade-start[101133]: dupeguru> ImportError: cannot import name 'sandbox' from 'setuptools' (/nix/store/63r62p69dpgzzh4iivn8zr26n9adysdl-python3.11-setuptools-80.7.1/lib/python3.11/site-packages/setuptools/__init__.py)
Jul 13 10:00:32 Odin nixos-upgrade-start[101133]: dupeguru> make: *** [Makefile:88: modules] Error 1
Jul 13 10:00:32 Odin nixos-upgrade-start[101133]: building '/nix/store/xylxq6j0axmhpjgkn4kbh21r1ybpdz1v-hm_.configplankdock1launchers18io.lmms.LMMS.dockitem.drv'...
Jul 13 10:00:32 Odin nixos-upgrade-start[101133]: error: builder for '/nix/store/3mlq8aklkv4f1bkns99br0r34zy2ixbn-dupeguru-4.3.1.drv' failed with exit code 2
Jul 13 10:00:32 Odin nixos-upgrade-start[101133]: error: 1 dependencies of derivation '/nix/store/fxzk4d3hbxxs7yb0jv7i0nykivlbg3k2-home-manager-path.drv' failed to build
Jul 13 10:00:32 Odin nixos-upgrade-start[101133]: error: 1 dependencies of derivation '/nix/store/xhjfxh9m3hyjdnqrw9mvw2fljqg6gmcs-home-manager-generation.drv' failed to build
Jul 13 10:00:32 Odin nixos-upgrade-start[101133]: error: 1 dependencies of derivation '/nix/store/390hkq3svxd2zcmkywx8pp74r8xsg9bj-unit-home-manager-beatlink.service.drv' failed to build
Jul 13 10:00:32 Odin nixos-upgrade-start[101133]: error: 1 dependencies of derivation '/nix/store/hmi7afaydp6v63kw951vypvs1qkq4r52-system-units.drv' failed to build
Jul 13 10:00:32 Odin nixos-upgrade-start[101133]: error: 1 dependencies of derivation '/nix/store/7k9xs0rha3n648x7kk2qfbwffz6a9fzr-etc.drv' failed to build
Jul 13 10:00:32 Odin nixos-upgrade-start[101133]: error: 1 dependencies of derivation '/nix/store/pwlrxnf96q61jx3pkas849x3c2d6cd7i-nixos-system-Odin-25.11.20250708.9807714.drv' failed to build
Job for nixos-upgrade.service failed because the control process exited with error code.
See "systemctl status nixos-upgrade.service" and "journalctl -xeu nixos-upgrade.service" for details.
Jul 13 10:00:33 Odin nixos-upgrade-start[101021]: Command 'nix --extra-experimental-features 'nix-command flakes' build --print-out-paths 'github:BeatLink/TechNet#nixosConfigurations."Odin".config.system.build.toplevel' --print-build-logs --refresh --no-write-lock-file --no-link' returned non-zero exit status 1.
Jul 13 10:00:33 Odin systemd[1]: nixos-upgrade.service: Main process exited, code=exited, status=1/FAILURE
```

### Additional context

ImportError: cannot import name 'sandbox' from 'setuptools' (/nix/store/y0iwy8ma2m45pr23bw4r7lyz69v2b6wf-python3.11-setuptools-80.7.1/lib/python3.11/site-packages/setuptools/__init__.py)

### System metadata

 - system: `"x86_64-linux"`
 - host os: `Linux 6.12.36, NixOS, 25.11 (Xantusia), 25.11.20250708.9807714`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.28.4`
 - nixpkgs: `/nix/store/bgl6ldj5ihbwcq8p42z3a0qzgqafgk2b-source`


### Notify maintainers

@novoxd 

---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+build+failure%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/applications/misc/dupeguru/default.nix
+++ b/pkgs/applications/misc/dupeguru/default.nix
@@ -20,6 +20,10 @@ python3Packages.buildPythonApplication rec {
     hash = "sha256-/jkZiCapmCLMp7WfgUmpsR8aNCfb3gBELlMYaC4e7zI=";
   };
 
+  patches = [
+    ./remove-setuptools-sandbox.patch
+  ];
+
   nativeBuildInputs = [
     gettext
     python3Packages.pyqt5

--- a/pkgs/applications/misc/dupeguru/remove-setuptools-sandbox.patch
+++ b/pkgs/applications/misc/dupeguru/remove-setuptools-sandbox.patch
@@ -1,0 +1,27 @@
+diff --git a/build.py b/build.py
+index 06905a11..56d54a17 100644
+--- a/build.py
++++ b/build.py
+@@ -10,7 +10,7 @@ from optparse import OptionParser
+ import shutil
+ from multiprocessing import Pool
+ 
+-from setuptools import sandbox
++import subprocess
+ from hscommon import sphinxgen
+ from hscommon.build import (
+     add_to_pythonpath,
+@@ -118,7 +118,12 @@ def build_normpo():
+ def build_pe_modules():
+     print("Building PE Modules")
+     # Leverage setup.py to build modules
+-    sandbox.run_setup("setup.py", ["build_ext", "--inplace"])
++    result = subprocess.run(
++        [sys.executable, "setup.py", "build_ext", "--inplace"],
++        check=True,
++    )
++    if result.returncode != 0:
++        sys.exit("Error building PE modules. Please check the output above.")
+ 
+ 
+ def build_normal():

